### PR TITLE
feat[experiment]: Test case to validate if the cspc pool creation is failed when the blockdevice is already claimed

### LINKED ISSUE #297: cStor CSPC pool creation has to fail when CSPC configured with already claimed blockdevices
- Admission server should be able to restrict pool creation

### PR DESCRIPTION
**Pull request by nsathyaseelan**
Signed-off-by: nsathyaseelan <sathyaseelan.n@mayadata.io>

 - In this test case trying to create the cspc pool when the block device is already in claimed state and validating the creation is blocked by admission server

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
* [x] Have you included relevant README for the chaoslib/experiment with details?
* [x] Have you added debug messages where necessary? 
* [x] Have you added task comments where necessary? 
* [x] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [x] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [x] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [x] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [x] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [x] Can you suggest the minimal resource requirements for the litmusbook execution?
* [x] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [x] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

```
No config file found; using defaults
/etc/ansible/hosts did not meet host_list requirements, check plugin documentation if this is unexpected
/etc/ansible/hosts did not meet script requirements, check plugin documentation if this is unexpected

PLAY [localhost] ***************************************************************
2020-07-29T06:37:43.499965 (delta: 0.038881)         elapsed: 0.038881 ******** 
=============================================================================== 

TASK [include_tasks] ***********************************************************
2020-07-29T06:37:43.509995 (delta: 0.009996)         elapsed: 0.048911 ******** 
included: /utils/fcm/create_testname.yml for 127.0.0.1

TASK [Record test instance/run ID] *********************************************
2020-07-29T06:37:43.534725 (delta: 0.024716)         elapsed: 0.073641 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Construct testname appended with runID] **********************************
2020-07-29T06:37:43.561721 (delta: 0.026959)         elapsed: 0.100637 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [include_tasks] ***********************************************************
2020-07-29T06:37:43.590745 (delta: 0.028993)         elapsed: 0.129661 ******** 
included: /utils/fcm/update_litmus_result_resource.yml for 127.0.0.1

TASK [Generate the litmus result CR to reflect SOT (Start of Test)] ************
2020-07-29T06:37:43.659536 (delta: 0.068682)         elapsed: 0.198452 ******** 
changed: [127.0.0.1] => {"changed": true, "checksum": "7b34b14a0de7745d1bf128c4a1d93cef06cacf22", "dest": "./litmus-result.yaml", "gid": 0, "group": "root", "md5sum": "4ec717e4d99a4c5df6d314da6255a836", "mode": "0644", "owner": "root", "size": 436, "src": "/root/.ansible/tmp/ansible-tmp-1596004663.7-180086901415747/source", "state": "file", "uid": 0}

TASK [Analyze the cr yaml] *****************************************************
2020-07-29T06:37:44.149341 (delta: 0.489778)         elapsed: 0.688257 ******** 
changed: [127.0.0.1] => {"changed": true, "cmd": "cat litmus-result.yaml", "delta": "0:00:00.558806", "end": "2020-07-29 06:37:44.954111", "rc": 0, "start": "2020-07-29 06:37:44.395305", "stderr": "", "stderr_lines": [], "stdout": "---\napiVersion: litmus.io/v1alpha1\nkind: LitmusResult\nmetadata:\n\n  # name of the litmus testcase\n  name: block-pool-creation-with-claimed-bd \nspec:\n\n  # holds information on the testcase\n  testMetadata:\n    app:  \n    chaostype:  \n\n  # holds the state of testcase,  manually updated by json merge patch\n  # result is the useful value today, but anticipate phase use in future \n  testStatus: \n    phase: in-progress  \n    result: none ", "stdout_lines": ["---", "apiVersion: litmus.io/v1alpha1", "kind: LitmusResult", "metadata:", "", "  # name of the litmus testcase", "  name: block-pool-creation-with-claimed-bd ", "spec:", "", "  # holds information on the testcase", "  testMetadata:", "    app:  ", "    chaostype:  ", "", "  # holds the state of testcase,  manually updated by json merge patch", "  # result is the useful value today, but anticipate phase use in future ", "  testStatus: ", "    phase: in-progress  ", "    result: none "]}

TASK [Apply the litmus result CR] **********************************************
2020-07-29T06:37:44.999424 (delta: 0.850028)         elapsed: 1.53834 ********* 
changed: [127.0.0.1] => {"changed": true, "failed_when_result": false, "method": "patch", "result": {"apiVersion": "litmus.io/v1alpha1", "kind": "LitmusResult", "metadata": {"creationTimestamp": "2020-07-28T15:13:53Z", "generation": 11, "name": "block-pool-creation-with-claimed-bd", "resourceVersion": "282392", "selfLink": "/apis/litmus.io/v1alpha1/litmusresults/block-pool-creation-with-claimed-bd", "uid": "4381715e-aab9-4971-882a-3542005ac9d6"}, "spec": {"testMetadata": {}, "testStatus": {"phase": "in-progress", "result": "none"}}}}

TASK [Generate the litmus result CR to reflect EOT (End of Test)] **************
2020-07-29T06:37:45.842735 (delta: 0.843285)         elapsed: 2.381651 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Analyze the cr yaml] *****************************************************
2020-07-29T06:37:45.869047 (delta: 0.026176)         elapsed: 2.407963 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Apply the litmus result CR] **********************************************
2020-07-29T06:37:45.893329 (delta: 0.024178)         elapsed: 2.432245 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Getting the compute node name] *******************************************
2020-07-29T06:37:45.920772 (delta: 0.027385)         elapsed: 2.459688 ******** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get nodes --no-headers | grep -v master | awk {'print $1'} | head -3", "delta": "0:00:00.730504", "end": "2020-07-29 06:37:46.775746", "failed_when_result": false, "rc": 0, "start": "2020-07-29 06:37:46.045242", "stderr": "", "stderr_lines": [], "stdout": "e2e1-node1\ne2e1-node2\ne2e1-node3", "stdout_lines": ["e2e1-node1", "e2e1-node2", "e2e1-node3"]}

TASK [set the value for the disk count to fetch the unclaimed blockDevice from each node] ***
2020-07-29T06:37:46.813773 (delta: 0.892934)         elapsed: 3.352689 ******** 
 [WARNING]: when statements should not include jinja2 templating delimiters
such as {{ }} or {% %}. Found: '{{ pool_type }}' in item.key
skipping: [127.0.0.1] => (item={'key': u'raidz2', 'value': {u'count': 6}})  => {"changed": false, "item": {"key": "raidz2", "value": {"count": 6}}, "skip_reason": "Conditional result was False"}
skipping: [127.0.0.1] => (item={'key': u'mirror', 'value': {u'count': 2}})  => {"changed": false, "item": {"key": "mirror", "value": {"count": 2}}, "skip_reason": "Conditional result was False"}
ok: [127.0.0.1] => (item={'key': u'stripe', 'value': {u'count': 1}}) => {"ansible_facts": {"disk_count": "1"}, "changed": false, "item": {"key": "stripe", "value": {"count": 1}}}
skipping: [127.0.0.1] => (item={'key': u'raidz', 'value': {u'count': 3}})  => {"changed": false, "item": {"key": "raidz", "value": {"count": 3}}, "skip_reason": "Conditional result was False"}

TASK [Add node labels for each nodes and create blockdevice template] **********
2020-07-29T06:37:46.907647 (delta: 0.093837)         elapsed: 3.446563 ******** 
changed: [127.0.0.1] => (item=[u'e2e1-node1']) => {"changed": true, "checksum": "852d03a32d4e2a17ebb7f60e57b64ccfc0ec328d", "dest": "./blockdevice-e2e1-node1.yml", "gid": 0, "group": "root", "item": ["e2e1-node1"], "md5sum": "df6632fefb3b3720d7676921d4c61ff0", "mode": "0644", "owner": "root", "size": 162, "src": "/root/.ansible/tmp/ansible-tmp-1596004666.95-107585839097993/source", "state": "file", "uid": 0}
changed: [127.0.0.1] => (item=[u'e2e1-node2']) => {"changed": true, "checksum": "40d1692102b850567b3a541548ca6dbb47de770b", "dest": "./blockdevice-e2e1-node2.yml", "gid": 0, "group": "root", "item": ["e2e1-node2"], "md5sum": "ccbf4731c17dacfae7a70cae582f6f38", "mode": "0644", "owner": "root", "size": 162, "src": "/root/.ansible/tmp/ansible-tmp-1596004667.19-254086062244953/source", "state": "file", "uid": 0}
changed: [127.0.0.1] => (item=[u'e2e1-node3']) => {"changed": true, "checksum": "55c2b6fed06fe0362b472b1d9e04b8e5499841ed", "dest": "./blockdevice-e2e1-node3.yml", "gid": 0, "group": "root", "item": ["e2e1-node3"], "md5sum": "f76670d47194b1470f18ec6b2afc92ff", "mode": "0644", "owner": "root", "size": 162, "src": "/root/.ansible/tmp/ansible-tmp-1596004667.43-242421739752896/source", "state": "file", "uid": 0}

TASK [Add the block devices for each node's block device template] *************
2020-07-29T06:37:47.660458 (delta: 0.75275)         elapsed: 4.199374 ********* 
included: /experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/add_blockdevice.yml for 127.0.0.1
included: /experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/add_blockdevice.yml for 127.0.0.1
included: /experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/add_blockdevice.yml for 127.0.0.1

TASK [Getting the Unclaimed block-device from each node] ***********************
2020-07-29T06:37:47.722206 (delta: 0.061718)         elapsed: 4.261122 ******** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get blockdevice -n openebs -l kubernetes.io/hostname=e2e1-node1 -o json | jq '.items[] | select(.status.claimState==\"Claimed\") | select(.status.state==\"Active\") | .metadata.name' | tr \"\\\"\" \" \" | grep -v sparse | head -n \"1\"", "delta": "0:00:00.646665", "end": "2020-07-29 06:37:48.523173", "rc": 0, "start": "2020-07-29 06:37:47.876508", "stderr": "", "stderr_lines": [], "stdout": " blockdevice-18e16d1ba63be85888cfcbf7e27a808b ", "stdout_lines": [" blockdevice-18e16d1ba63be85888cfcbf7e27a808b "]}

TASK [Add the block devices] ***************************************************
2020-07-29T06:37:48.560665 (delta: 0.838431)         elapsed: 5.099581 ******** 
changed: [127.0.0.1] => (item= blockdevice-18e16d1ba63be85888cfcbf7e27a808b ) => {"backup": "", "changed": true, "item": " blockdevice-18e16d1ba63be85888cfcbf7e27a808b ", "msg": "line added"}

TASK [Getting the Unclaimed block-device from each node] ***********************
2020-07-29T06:37:48.846638 (delta: 0.285931)         elapsed: 5.385554 ******** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get blockdevice -n openebs -l kubernetes.io/hostname=e2e1-node2 -o json | jq '.items[] | select(.status.claimState==\"Claimed\") | select(.status.state==\"Active\") | .metadata.name' | tr \"\\\"\" \" \" | grep -v sparse | head -n \"1\"", "delta": "0:00:00.699215", "end": "2020-07-29 06:37:49.675446", "rc": 0, "start": "2020-07-29 06:37:48.976231", "stderr": "", "stderr_lines": [], "stdout": " blockdevice-609310ea5c4c91f20a781ca4f0a52330 ", "stdout_lines": [" blockdevice-609310ea5c4c91f20a781ca4f0a52330 "]}

TASK [Add the block devices] ***************************************************
2020-07-29T06:37:49.706617 (delta: 0.859945)         elapsed: 6.245533 ******** 
changed: [127.0.0.1] => (item= blockdevice-609310ea5c4c91f20a781ca4f0a52330 ) => {"backup": "", "changed": true, "item": " blockdevice-609310ea5c4c91f20a781ca4f0a52330 ", "msg": "line added"}

TASK [Getting the Unclaimed block-device from each node] ***********************
2020-07-29T06:37:49.877799 (delta: 0.170993)         elapsed: 6.416715 ******** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl get blockdevice -n openebs -l kubernetes.io/hostname=e2e1-node3 -o json | jq '.items[] | select(.status.claimState==\"Claimed\") | select(.status.state==\"Active\") | .metadata.name' | tr \"\\\"\" \" \" | grep -v sparse | head -n \"1\"", "delta": "0:00:00.646967", "end": "2020-07-29 06:37:50.660131", "rc": 0, "start": "2020-07-29 06:37:50.013164", "stderr": "", "stderr_lines": [], "stdout": " blockdevice-3f2d933fd9573945308218724d25f777 ", "stdout_lines": [" blockdevice-3f2d933fd9573945308218724d25f777 "]}

TASK [Add the block devices] ***************************************************
2020-07-29T06:37:50.689242 (delta: 0.811324)         elapsed: 7.228158 ******** 
changed: [127.0.0.1] => (item= blockdevice-3f2d933fd9573945308218724d25f777 ) => {"backup": "", "changed": true, "item": " blockdevice-3f2d933fd9573945308218724d25f777 ", "msg": "line added"}

TASK [Include the blockdevice template in the CSPC spec] ***********************
2020-07-29T06:37:50.862479 (delta: 0.173207)         elapsed: 7.401395 ******** 
changed: [127.0.0.1] => (item=e2e1-node1) => {"changed": true, "item": "e2e1-node1", "msg": "Block inserted"}
changed: [127.0.0.1] => (item=e2e1-node2) => {"changed": true, "item": "e2e1-node2", "msg": "Block inserted"}
changed: [127.0.0.1] => (item=e2e1-node3) => {"changed": true, "item": "e2e1-node3", "msg": "Block inserted"}

TASK [Replacing the pool name in CSPC spec] ************************************
2020-07-29T06:37:51.332727 (delta: 0.47022)         elapsed: 7.871643 ********* 
changed: [127.0.0.1] => {"changed": true, "msg": "1 replacements made"}

TASK [Replacing the namespace in CSPC spec] ************************************
2020-07-29T06:37:51.579375 (delta: 0.246598)         elapsed: 8.118291 ******** 
changed: [127.0.0.1] => {"changed": true, "msg": "1 replacements made"}

TASK [Replacing the pool type in CSPC spec] ************************************
2020-07-29T06:37:51.745753 (delta: 0.166322)         elapsed: 8.284669 ******** 
changed: [127.0.0.1] => {"changed": true, "msg": "3 replacements made"}

TASK [Display cspc.yml for verification] ***************************************
2020-07-29T06:37:51.919331 (delta: 0.173481)         elapsed: 8.458247 ******** 
ok: [127.0.0.1] => (item=apiVersion: cstor.openebs.io/v1
kind: CStorPoolCluster
metadata:
  name: cstor-cspc-disk-pool-claimed
  namespace: openebs
spec:
  pools:
# BEGIN ANSIBLE MANAGED BLOCK
    - nodeSelector:
        kubernetes.io/hostname: e2e1-node3
      dataRaidGroups:
      - blockDevices:
        - blockDeviceName:  blockdevice-3f2d933fd9573945308218724d25f777 
      poolConfig:
        dataRaidGroupType: stripe
# ## e2e1-node3 Ansible Config ## ANSIBLE MANAGED BLOCK
# BEGIN ANSIBLE MANAGED BLOCK
    - nodeSelector:
        kubernetes.io/hostname: e2e1-node2
      dataRaidGroups:
      - blockDevices:
        - blockDeviceName:  blockdevice-609310ea5c4c91f20a781ca4f0a52330 
      poolConfig:
        dataRaidGroupType: stripe
# ## e2e1-node2 Ansible Config ## ANSIBLE MANAGED BLOCK
# BEGIN ANSIBLE MANAGED BLOCK
    - nodeSelector:
        kubernetes.io/hostname: e2e1-node1
      dataRaidGroups:
      - blockDevices:
        - blockDeviceName:  blockdevice-18e16d1ba63be85888cfcbf7e27a808b 
      poolConfig:
        dataRaidGroupType: stripe
# ## e2e1-node1 Ansible Config ## ANSIBLE MANAGED BLOCK) => {
    "item": "apiVersion: cstor.openebs.io/v1\nkind: CStorPoolCluster\nmetadata:\n  name: cstor-cspc-disk-pool-claimed\n  namespace: openebs\nspec:\n  pools:\n# BEGIN ANSIBLE MANAGED BLOCK\n    - nodeSelector:\n        kubernetes.io/hostname: e2e1-node3\n      dataRaidGroups:\n      - blockDevices:\n        - blockDeviceName:  blockdevice-3f2d933fd9573945308218724d25f777 \n      poolConfig:\n        dataRaidGroupType: stripe\n# ## e2e1-node3 Ansible Config ## ANSIBLE MANAGED BLOCK\n# BEGIN ANSIBLE MANAGED BLOCK\n    - nodeSelector:\n        kubernetes.io/hostname: e2e1-node2\n      dataRaidGroups:\n      - blockDevices:\n        - blockDeviceName:  blockdevice-609310ea5c4c91f20a781ca4f0a52330 \n      poolConfig:\n        dataRaidGroupType: stripe\n# ## e2e1-node2 Ansible Config ## ANSIBLE MANAGED BLOCK\n# BEGIN ANSIBLE MANAGED BLOCK\n    - nodeSelector:\n        kubernetes.io/hostname: e2e1-node1\n      dataRaidGroups:\n      - blockDevices:\n        - blockDeviceName:  blockdevice-18e16d1ba63be85888cfcbf7e27a808b \n      poolConfig:\n        dataRaidGroupType: stripe\n# ## e2e1-node1 Ansible Config ## ANSIBLE MANAGED BLOCK"
}

TASK [Create cstor disk pool] **************************************************
2020-07-29T06:37:51.979191 (delta: 0.059834)         elapsed: 8.518107 ******** 
changed: [127.0.0.1] => {"changed": true, "cmd": "kubectl apply -f cspc.yml", "delta": "0:00:00.874137", "end": "2020-07-29 06:37:52.986136", "failed_when_result": false, "msg": "non-zero return code", "rc": 1, "start": "2020-07-29 06:37:52.111999", "stderr": "Error from server (BadRequest): error when creating \"cspc.yml\": admission webhook \"admission-webhook.cstor.openebs.io\" denied the request: invalid cspc specification: invalid pool spec: error: can't use claimed blockdevice blockdevice-3f2d933fd9573945308218724d25f777", "stderr_lines": ["Error from server (BadRequest): error when creating \"cspc.yml\": admission webhook \"admission-webhook.cstor.openebs.io\" denied the request: invalid cspc specification: invalid pool spec: error: can't use claimed blockdevice blockdevice-3f2d933fd9573945308218724d25f777"], "stdout": "", "stdout_lines": []}

TASK [set_fact] ****************************************************************
2020-07-29T06:37:53.015475 (delta: 1.036255)         elapsed: 9.554391 ******** 
ok: [127.0.0.1] => {"ansible_facts": {"flag": "Pass"}, "changed": false}

TASK [include_tasks] ***********************************************************
2020-07-29T06:37:53.058212 (delta: 0.042697)         elapsed: 9.597128 ******** 
included: /utils/fcm/update_litmus_result_resource.yml for 127.0.0.1

TASK [Generate the litmus result CR to reflect SOT (Start of Test)] ************
2020-07-29T06:37:53.094800 (delta: 0.036534)         elapsed: 9.633716 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Analyze the cr yaml] *****************************************************
2020-07-29T06:37:53.123062 (delta: 0.028241)         elapsed: 9.661978 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Apply the litmus result CR] **********************************************
2020-07-29T06:37:53.143723 (delta: 0.020609)         elapsed: 9.682639 ******** 
skipping: [127.0.0.1] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Generate the litmus result CR to reflect EOT (End of Test)] **************
2020-07-29T06:37:53.164336 (delta: 0.020588)         elapsed: 9.703252 ******** 
changed: [127.0.0.1] => {"changed": true, "checksum": "206ba6559e5bc8050d9520e23d49af16e69207db", "dest": "./litmus-result.yaml", "gid": 0, "group": "root", "md5sum": "51925c4883a58cce58d7c21a0af10e4e", "mode": "0644", "owner": "root", "size": 434, "src": "/root/.ansible/tmp/ansible-tmp-1596004673.2-154666082393014/source", "state": "file", "uid": 0}

TASK [Analyze the cr yaml] *****************************************************
2020-07-29T06:37:54.373349 (delta: 1.208988)         elapsed: 10.912265 ******* 
changed: [127.0.0.1] => {"changed": true, "cmd": "cat litmus-result.yaml", "delta": "0:00:00.506560", "end": "2020-07-29 06:37:55.000360", "rc": 0, "start": "2020-07-29 06:37:54.493800", "stderr": "", "stderr_lines": [], "stdout": "---\napiVersion: litmus.io/v1alpha1\nkind: LitmusResult\nmetadata:\n\n  # name of the litmus testcase\n  name: block-pool-creation-with-claimed-bd \nspec:\n\n  # holds information on the testcase\n  testMetadata:\n    app:  \n    chaostype:  \n\n  # holds the state of testcase,  manually updated by json merge patch\n  # result is the useful value today, but anticipate phase use in future \n  testStatus: \n    phase: completed  \n    result: Pass ", "stdout_lines": ["---", "apiVersion: litmus.io/v1alpha1", "kind: LitmusResult", "metadata:", "", "  # name of the litmus testcase", "  name: block-pool-creation-with-claimed-bd ", "spec:", "", "  # holds information on the testcase", "  testMetadata:", "    app:  ", "    chaostype:  ", "", "  # holds the state of testcase,  manually updated by json merge patch", "  # result is the useful value today, but anticipate phase use in future ", "  testStatus: ", "    phase: completed  ", "    result: Pass "]}

TASK [Apply the litmus result CR] **********************************************
2020-07-29T06:37:55.026897 (delta: 0.65352)         elapsed: 11.565813 ******** 
changed: [127.0.0.1] => {"changed": true, "failed_when_result": false, "method": "patch", "result": {"apiVersion": "litmus.io/v1alpha1", "kind": "LitmusResult", "metadata": {"creationTimestamp": "2020-07-28T15:13:53Z", "generation": 12, "name": "block-pool-creation-with-claimed-bd", "resourceVersion": "282436", "selfLink": "/apis/litmus.io/v1alpha1/litmusresults/block-pool-creation-with-claimed-bd", "uid": "4381715e-aab9-4971-882a-3542005ac9d6"}, "spec": {"testMetadata": {}, "testStatus": {"phase": "completed", "result": "Pass"}}}}

PLAY RECAP *********************************************************************
127.0.0.1                  : ok=28   changed=19   unreachable=0    failed=0   

2020-07-29T06:37:55.615132 (delta: 0.588207)         elapsed: 12.154048 ******* 
=============================================================================== 
```

### CODE REVIEW

File: experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Test Case to validate the cspc pool creation is failed when the bloxkdevice is already claimed.
+
+## Description
+   - This test is capable of validate the pool creation is failed when the blockdevice is claimed.
```
Comment (gprasath) [inline experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/README.md]: This test is capable of validating if the pool creation is failed when the specified blockdevices are already claimed

File: experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/run_litmus_test.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: block-pool-creation-with-claimed-bd-
```
Comment (gprasath) [inline experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/run_litmus_test.yml]: change it to `cstor-pool-creation-with-claimed-bd-`

File: experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/run_litmus_test.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: block-pool-creation-with-claimed-bd
```
Comment (gprasath) [inline experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/run_litmus_test.yml]: change this as well

File: experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/add_blockdevice.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+    - name: Getting the Unclaimed block-device from each node
```
Comment (gprasath) [inline experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/add_blockdevice.yml]: we are getting claimed block devices. Change the task name accordingly

File: experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Test Case to validate the cspc pool creation is failed when the bloxkdevice is already claimed.
```
Comment (w3aman) [inline experiments/functional/cspc-pool/block-pool-creation-with-claimed-bd/README.md]: blockdevice**

**Review by w3aman — COMMENTED** [review]
/looks good from code side

**Review by gprasath — APPROVED** [review]

**Pull request merged**